### ISSUE TITLE
Change aspect ratio of game

### ISSUE THREAD

**Issue by Danimal696** [OpenDungeons/OpenDungeons#1180]
We have pretty nice models, but so far they are being wasted by having everything into tiny corridors and weirdly sized walls.
im posting a few links of what i think some proper ratio:
http://www.theaveragegamer.com/wp-content/uploads/2015/03/War-for-the-Overworld-Overhead.jpg
http://core0.staticworld.net/images/article/2015/04/2015-04-19_00032-100580950-orig.jpg

Leaving aside the graphical quality we cant match yet, and resuming, i would say, bigger tiles, smaller models.
What would be the main problems against this?

**Comment by Danimal696** [comment]
I think ill try my hand at this in the future, @hwoarangmy sometime ago i asked you if you could upload a cleaner version of the tiles including only the used ones, did you upload it? if not, could you do it and name it to something easy to spot?.

**Comment by hwoarangmy** [comment]
If you want, I will build you a snaphot allowing to tune the size of creatures. If you find an aspect ratio that looks good, we can adapt the camera and resize all.

**Comment by Danimal696** [comment]
i think allowing to resize the tile sizes would be better, btw, do you still have the cleaned up tileset file?

**Comment by hwoarangmy** [comment]
> i think allowing to resize the tile sizes would be better

No. The tiles are our unit size. Changing creature size or tile size is equivalent (increasing tiles is equivalent to decreasing everything else). But if we change tile size, we have to adapt every position (otherwise, tiles will not be connected).
If we can achieve a fitting size for creatures, we can scale them and after, if needed, scale room objects.

> btw, do you still have the cleaned up tileset file?

The blend file ?
If yes, it is in the repo AFAIK

**Comment by hwoarangmy** [comment]
@Danimal696 I've uploaded the version:
http://download.tuxfamily.org/opendungeons/WIP/OpenDungeons-git20160601-Win32-MSVS2013-resize.zip

There is a dirty hack allowing to rescale creatures. To use it, go to config/global.cfg and check the last parameter in the file. By default, it is set to 0.5.

**Comment by Danimal696** [comment]
Thanks @hwoarangmy , this is something that will need a lot of thinkering, but if i manage we can increase graphical quality a lot, make zooming stronger and smaller maps look good.

**Comment by hwoarangmy** [comment]
Note that only creatures will be resized. Building objects, default camera position and max/min camera height can be adapted later once the aspect ratio is good.

**Comment by Danimal696** [comment]
go figures... now when i export the tileset all models are too big, am i forgetting something? i remember that at least the walls i remade were ready-to-go. Have there been any other scale change since i made those?

**Comment by hwoarangmy** [comment]
yep. Now, there is no ingame scaling anymore. To update a model, you need to scale it in the mesh file. That will avoid weird issues like weapons  becoming too small/big if you use them in a model not scaled as expected.

More importantly, to know how to resize correctly a model (be it a room object, a creature or a tile), you can have a look at the wiki:
https://github.com/OpenDungeons/OpenDungeons/wiki/Adding-a-model-in-the-game

As said in the wiki, the tools can be downloaded here:
http://download.tuxfamily.org/opendungeons/Tools/OgreMeshMagick.zip

IIRC, the old tileset was resized 0.4, 0.4, 0.5.
So, once you get the mesh file, you can resize it using Ogre tools in the above link:

```
OgreMeshMagick transform -scale=0.4/0.4/0.5 Tile.mesh
```

**Comment by Danimal696** [comment]
testing with new values 0.75 seems a very good new ratio, creatures seem to be inside a dungeon instead of poking over it

**Comment by hwoarangmy** [comment]
Could you please post some snapshots so that we can have a look ?

**Comment by Danimal696** [comment]
At first it looks strange, and many objects need resizing as well, but now the creatures fit were they are supposed to live (Skeletons and lava spawn need further resizing as well, im ok if lava spawn is that big, and i already found a great replacement for lich so no worries there):
![opendungeons 2016-06-15 16-22-35-501](https://cloud.githubusercontent.com/assets/8299756/16083662/1f39d802-3316-11e6-82cb-587ac0cb1f6a.jpg)
![opendungeons 2016-06-15 16-18-34-728](https://cloud.githubusercontent.com/assets/8299756/16083663/1f41b874-3316-11e6-94a9-ee46cb6f951d.jpg)
![opendungeons 2016-06-15 16-19-24-267](https://cloud.githubusercontent.com/assets/8299756/16083664/1f4a0736-3316-11e6-89f3-e3239f240542.jpg)
![opendungeons 2016-06-15 16-20-04-788](https://cloud.githubusercontent.com/assets/8299756/16083665/1f4acb1c-3316-11e6-8fca-b46e88d1275f.jpg)

**Comment by hwoarangmy** [comment]
and what size would you like lava spawn to be ?

**Comment by Danimal696** [comment]
its fine for now since i asume its a tier 3 or so creature

**Comment by hwoarangmy** [comment]
ok. Anyway, we can always change its size when we get better models

**Comment by Loobinex** [comment]
Be sure to also modify the scroll speed when you're zoomed in more, which is what you'll do more often now if you want to see your creatures. (#1142)
